aliases/available/general.aliases.bash

```diff
@@ -15,9 +15,14 @@ alias la='ls -AF'       # Compact view, show hidden
 alias ll='ls -al'
 alias l='ls -a'
 alias l1='ls -1'
+alias lisah='ls -lisah --color=auto'
 
 alias _="sudo"
 
+function mkdirr() {
```

Comment:
This shouldn't be in an `alias` file - please move this to the `base` plugin.

Reply:
https://github.com/jusdepatate/bash-it/commit/962f3093a2fa8c364e72d8e970ce7e349e8caa54